backend/repository/config.go

```diff
@@ -65,12 +65,43 @@ type EndpointConfig struct {
 	WorkflowType     string                    `json:"workflowType"`
 	ClientID         string                    `json:"clientID"`
 	ClientMethod     string                    `json:"clientMethod"`
-	TestFixtures     []string                  `json:"testFixtures"`
+	TestFixtures     []*TestStub                  `json:"testFixtures"`
 	Middlewares      []*codegen.MiddlewareSpec `json:"middlewares"`
 	ReqHeaderMap     map[string]string         `json:"reqHeaderMap"`
 	ResHeaderMap     map[string]string         `json:"resHeaderMap"`
 }
 
+// TestStub saves stubbed requests/responses for an endpoint test.
+type TestStub struct {
+	TestName               string `json:"testName"`
+	EndpointID             string `json:"endpointId"`
+	HandlerID              string `json:"handlerId"`
+	EndpointRequest        map[string]interface{} `json:"endpointRequest"`
```

Comment:
Seems that you can define `Request` type for both Endpoint and client, including the json body/ string body and header.

Reply:
put this struct here not because it makes completely sense but since it's used in module system, we might(will have to) need to refactor how test fixtures are used & stored